src/SOS/SOS.Hosting/CorDebugDataTargetWrapper.cs

```diff
@@ -107,6 +107,9 @@ protected override void Destroy()
                     case Architecture.Arm64:
                         platform = CorDebugPlatform.CORDB_PLATFORM_POSIX_ARM64;
                         break;
+                    case (Architecture)9 /* Architecture.RiscV64 */:
```

Comment:
As shown in the link below, RiscV64 seems to be included in the Architecture enum.
https://github.com/dotnet/runtime/blob/952ac21a0e89569c923f8597a0441138e3ddf89a/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs#L59

Reply:
Unfortunately, we could use it only after next (9.0?) runtime release. Current SDK release and even SDK 8.0 RC2 don't include it: https://github.com/dotnet/runtime/blob/v8.0.0-rc.2.23479.6/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs

Reply:
Thanks :)